contrib/vcl/source/BUILD.maybe

```diff
@@ -0,0 +1,132 @@
+load(
```

Comment:
Are the contrib/vcl/... changes needed in this PR?

Reply:
no! good spot! - ill remove - its an artifact of how i resolved conflict